src/ansi-c/expr2c.cpp

```diff
@@ -746,8 +746,8 @@ std::string expr2ct::convert_struct_type(
 /// To generate a C-like type declaration of an array. Includes the size of the
 /// array in the []
 /// \param src: The array type to convert
-/// qualifier
-/// declarator_str
+/// \param qualifiers: Any qualifiers on the type
+/// \param declarator_str: The array declarator
```

Comment:
Hm, these comments will shut up doxygen but I'm not sure they're really improving the documentation here.

Reply:
The main point of this PR is to shut up Doxygen.  We've set Doxygen to give warnings if you document some parameters and not others, and sometimes that means that we have documentation for parameters which is not especially helpful.

Reply:
How about wording this as "The previously computed string denoting the array symbol" - if that's any clearer @hannes-steffenhagen-diffblue?